### PR TITLE
migrate missing fa-external-link icon to fa-external-link-alt

### PR DESCRIPTION
**Pull request by bradymiller**

### CODE REVIEW

File: Documentation/help_files/report_dashboard_help.php [hunk 1]
```diff
@@ -245,7 +245,7 @@
                     <p><i class="fa fa-exclamation-triangle oe-text-red"  aria-hidden="true"></i> <strong><?php echo xlt("You will need Administrator privileges to setup phiMail Direct Messaging");?>.</strong></p>
 
                     <p><?php echo xlt("The first step is signing up for a production Direct messaging account with EMR Direct by registering on their website");?>.
-                        <a href="https://www.emrdirect.com/subscribe"" rel="noopener" target="_blank"><i class="fa fa-external-link text-primary" aria-hidden="true" data-original-title="" title=""></i></a>
+                        <a href="https://www.emrdirect.com/subscribe"" rel="noopener" target="_blank"><i class="fa fa-external-link-alt text-primary" aria-hidden="true" data-original-title="" title=""></i></a>
```
Comment (stu01509) [inline Documentation/help_files/report_dashboard_help.php:248]: ` <a href="https://www.emrdirect.com/subscribe""` unneccessary double quotes.

Comment (stephenwaite) [reply Documentation/help_files/report_dashboard_help.php:248]: nice catch @stu01509

Comment (bradymiller) [reply Documentation/help_files/report_dashboard_help.php:248]: eagle eyes :)

**Comment by bradymiller** [comment]
@bradymiller , code looks good and bringing in. thanks for the fix! -brady
@stu01509 , thanks for the review!

**Pull request merged**